examples/samples/autoForward.ts

```diff
@@ -0,0 +1,73 @@
+import { ReceiveMode, Namespace, generateUuid, SendableMessageInfo } from "../../lib";
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const str = process.env.SERVICEBUS_CONNECTION_STRING || "";
+const topic = process.env.TOPIC_NAME || "";
+const path = process.env.QUEUE_NAME || "";
+console.log("str: ", str);
+console.log("topic: ", topic);
+console.log("path: ", path);
+
+let ns: Namespace;
+
+// You need to setup a queue and a topic
+// The topic needs to have a subcription that will forward the messages from the topic to the queue
+
+// The sample assumes the topoloy described here is already setup.
+
```

Comment:
topoloy ?

Reply:
Learning to spell a lot of hard words!!! :)